docs/eventing/getting-started.md

```diff
@@ -0,0 +1,496 @@
+---
+title: "Getting Started with Eventing"
+linkTitle: "Getting Started"
+weight: 9
+type: "docs"
+---
+
+Use this guide to learn how to create, send, and verify events in Knative. The steps in this guide demonstrate a basic developer flow for managing events in Knative, including:
+
+- [Installing the Knative Eventing component](#installing-knative-eventing)
+- [Creating and configuring Knative Eventing Resources](#setting-up-knative-eventing-resources)
+- [Sending events with HTTP requests](#sending-events-to-the-broker)
+- [Verifying events were sent correctly](#verifying-events-were-received)
+
+## Before you begin
+
+To complete this guide, you will need the following installed and running:
+
+- A [Kubernetes cluster](https://kubernetes.io/docs/concepts/cluster-administration/cluster-administration-overview/) running v1.11 or higher
+- [Kubectl CLI tool](https://kubernetes.io/docs/reference/kubectl/overview/) v1.10 or higher
+- [curl v7.65 or higher](https://curl.haxx.se/download.html)
+- Knative Eventing Component
+
+### Installing Knative Eventing 
+
+If you previously [created a Knative cluster](../install/_index.md), you might already have Knative Eventing installed and running. You can check to see if the Eventing component exists on your cluster by running:
+
+```sh
+kubectl get pods --namespace knative-eventing
+```
+
+If the `knative-eventing` namespace does not exist, use the follow steps to add the Knative Eventing component to your cluster:
```

Comment:
```suggestion
Result:

<code>shell
NAME                                            READY   STATUS    RESTARTS   AGE
eventing-controller-6cdf6ff785-44k98            1/1     Running   0          29m
eventing-webhook-6c4f6699d8-qclbx               1/1     Running   0          29m
imc-controller-85cdb4946b-h2msj                 1/1     Running   0          13m
imc-dispatcher-5f4689d868-fspt6                 1/1     Running   0          13m
in-memory-channel-controller-d9db9d879-f2jw6    1/1     Running   0          13m
in-memory-channel-dispatcher-79bc7f46cd-4mvj6   1/1     Running   0          13m
sources-controller-5847564f4f-z59xc             1/1     Running   0          29m
</code>

If the `knative-eventing` namespace does not exist or the `in-memory-channel-controller-*` is not listed, use the following steps to install Knative Eventing with the in-memory channel provisioner:
```

NOTE: replace the `<code>` tags with markdown code blocks (the inline editor does not like the nested tick marks)